TestTools/StreamChatTestMockServer/Fixtures/JSONs/ws_health_check.json

```diff
@@ -1,44 +1,2953 @@
 {
-  "created_at" : "2021-04-23T06:05:31.659292606Z",
-  "me" : {
-    "id" : "luke_skywalker",
-    "banned" : false,
-    "roles" : [],
-    "unread_channels" : 0,
-    "mutes" : [],
-    "last_active" : "2021-04-23T06:05:31.656574549Z",
-    "created_at" : "2020-12-07T11:36:47.059906Z",
-    "devices" : [
-      {
-        "disabled" : true,
-        "id" : "e72f903a3ef67e279bea2da9565ede048e63a75223d3b592ee80612aa8a2e249",
-        "push_provider" : "apn",
-        "disabled_reason" : "BadDeviceToken",
-        "user_id" : "luke_skywalker",
-        "created_at" : "2020-12-17T11:33:46.940946Z"
-      },
-      {
-        "disabled" : true,
-        "id" : "60729e8d570b72beb7a1d2072530b9f7ab4a7292f2ea145e0671186111410bab",
-        "push_provider" : "apn",
-        "disabled_reason" : "BadDeviceToken",
-        "user_id" : "luke_skywalker",
-        "created_at" : "2020-12-17T11:10:00.204968Z"
-      }
-    ],
-    "invisible" : false,
-    "unread_count" : 0,
-    "channel_mutes" : [],
-    "image" : ".\/static\/media\/photo-1541271696563-3be2f555fc4e.4e84f363.jpeg",
-    "updated_at" : "2021-04-22T18:57:47.14206Z",
-    "language" : "",
-    "role" : "admin",
-    "total_unread_count" : 0,
-    "online" : true,
-    "name" : "luke_skywalker"
-  },
-  "type" : "health.check",
-  "connection_id" : "60782eca-0a05-154b-0000-000000a85747",
-  "cid" : "*"
-}
-
+    "cid": "*",
+    "connection_id": "6315fc7c-0a05-4049-0000-00001647dac7",
+    "created_at": "2022-12-01T12:56:41.765102666Z",
+    "me": {
+        "banned": false,
+        "birthland": "Tatooine",
+        "channel_mutes": [
+            {
+                "channel": {
```

Comment:
Why the ws_health_check is so big? Shouldn't this be split up between multiple events?

Reply:
Yeah, it is huge. This is basically what you get when connecting to a web socket in our DemoApp by luke_skywalker. I could parse its content on the fly and remove everything but the first channel, but honestly do not know if it's worth it.

Reply:
Hey @nuno-vieira, check out my latest commit ([9b6187](https://github.com/GetStream/stream-chat-swift/pull/2394/commits/9b6187f869063256971f0e42ce932272d975d52d)), I've reduced the size of both `ws_health_check` and `http_channels`.

Hope the tests will pass 😄

Reply:
@alteral If the response actually comes exactly like this, we can leave it. It's probably not worth adding more complexity. So I guess we can revert that change, if you want

Reply:
No, tbh, I like the idea of reducing those JSON templates. We are overwriting these objects in the tests anyway. Now it will be easier to review automated PRs.